### PR TITLE
help-activity

### PR DESCRIPTION
**Pull request by nwolisaemmanuel2-zz**
Update

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,23 +1,23 @@
 ![help-activity-icon](images/Activity-help.svg)
 
-Help is an activity which aims to provide users with easy access to documentation and manuals
+Help is an activity which aims to provide users with easy access to documentation and manual.
```
Comment (quozl) [inline README.md:3]: No, _manuals_ is correct, as the aim is to link Help to multiple other manuals.

File: README.md [hunk 2]
```diff
@@ -1,23 +1,23 @@
...
 
-Help can be accesed
+Help can be accesed.
```
Comment (quozl) [inline README.md:5]: No, there should not be a full stop here; it is a line that introduces a list.

File: README.md [hunk 3]
```diff
@@ -1,23 +1,23 @@
...
 * as a [standalone activity](http://activities.sugarlabs.org/en-US/sugar/addon/4051)
 * from [within another activity](#help-from-within-another-activity)
 * at https://help.sugarlabs.org/
 
-### How to edit Help and Contribute:
+### How to edit, Help and Contribute:
```
Comment (quozl) [inline README.md:10]: No, in this context "Help" is the name of the activity, so a comma is not correct.

File: README.md [hunk 4]
```diff
@@ -1,23 +1,23 @@
...
 [Read here](source/how_to_help.rst)
 
 ### Help from within another activity:
 
-When contextual help( <kbd>alt</kbd> + <kbd>shift</kbd> + <kbd>h</kbd> ) is requested from within an sugar activity, the [helplink.json](helplink.json) file matches the bundle_id(the same as defined in the particular activity.info file) with a corresponding html page and renders its content in a tiny browser.
+When contextual help( <kbd>alt</kbd> + <kbd>shift</kbd> + <kbd>h</kbd> ) is requested from within an sugar activity, the [helplink.json](helplink.json) file matches the bundle_id(the same as defined in the particular activity.info file) with a corresponding html pages and renders its content in a tiny browser.
```
Comment (quozl) [inline README.md:15]: This change is not correct.  The json file does link to a single page.

File: README.md [hunk 5]
```diff
@@ -1,23 +1,23 @@
...
 
 ### How to translate:
 https://github.com/godiard/help-activity/blob/master/howto_translate.rst
 
-###### More information at
+###### More information to be found at
```
Comment (quozl) [inline README.md:20]: Change is unnecessary.

**Review by quozl — CHANGES_REQUESTED** [review]
Thanks.  Good start, but I'm not sure English is your first language.

Also, please avoid editing files using GitHub, as we get multiple notifications and meaningless commit messages.  Please see our guide to [commit messages](https://github.com/sugarlabs/sugar-docs/blob/master/src/contributing.md#making-commits).

**Comment by chimosky** [comment]
@nwolisaemmanuel2 do you plan on completing this?

**Comment by chimosky** [comment]
Stale, closing.

**Pull request closed without merging**